### PR TITLE
Updates to the Algorithms Section

### PR DESCRIPTION
**Pull request by MandyW0312**
Just a few updates in the lecture sections of the introduction, day two, and day four

### CODE REVIEW

File: lessons/algorithms-day-two/lecture.md [hunk 1]
```diff
@@ -429,7 +429,7 @@ Middle is `21` and `25`, so median is: `23`
 
 [.build-lists: true]
 
-# 0, 1, few, $$ \infty $$
+# 0, 1, few, $$ \infinity $$
```
Comment (gstark) [inline lessons/algorithms-day-two/lecture.md:432]: This is a special formatting code and \infty is correct here

File: lessons/algorithms-day-two/lecture.md [hunk 2]
```diff
@@ -440,13 +440,13 @@ Middle is `21` and `25`, so median is: `23`
 
 ---
 
-# [fit] Use "0, 1, few, $$ \infty $$" in your thinking process
+# [fit] Use "0, 1, few, $$ \infinity $$" in your thinking process
```
Comment (gstark) [inline lessons/algorithms-day-two/lecture.md:443]: `\infty` is a special code and is correct here

File: lessons/algorithms-day-two/lecture.md [hunk 3]
```diff
@@ -440,13 +440,13 @@ Middle is `21` and `25`, so median is: `23`
...
 
 - Perhaps start with an input size of "a few."
 - Use your intuition to understand the problem
 - Don't forget about `0` and `1` inputs
   - (sometimes the problem has a curveball here)
-- See if your intuition still works with $$ \infty $$
```
Comment (gstark) [inline lessons/algorithms-day-two/lecture.md:449]: Same, `\infty` is a special code.

File: lessons/algorithms-day-two/lecture.md [hunk 4]
```diff
@@ -468,7 +468,7 @@ _"If something is true"_ or _"If something is false"_ help determine if or when
 # [fit] Loop with counting
 
 - "Do the following X times"
-- "Do the following until X is less-then/more-than/equal to Y"
```
Comment (gstark) [inline lessons/algorithms-day-two/lecture.md:471]: Same, `\infty` is a special code.

**Comment by gstark** [comment]
@MandyW0312 Undo the changes for `\infinity` and then we can merge this.

**Comment by MandyW0312** [comment]
@gstark Updated those changes back to \infty

**Pull request merged**